api.js

```diff
@@ -4,8 +4,12 @@ const ncNewsApi = axios.create({
   baseURL: "https://nc-news-1l6p.onrender.com/api",
 });
 
-export const getArticles = () => {
-  return ncNewsApi.get("/articles").then(({ data: { articles } }) => articles);
+export const getArticles = (topic) => {
+  if (topic) {
+    return ncNewsApi.get(`/articles?topic=${topic}`).then(({ data: { articles } }) => articles)
```

Comment:
Axios can do this for you - https://notes.northcoders.com/courses/js-front-end/axios#axios-params